plugins/services/src/js/data/ServicePlansClient.ts

```diff
@@ -10,7 +10,8 @@ export type ServicePlanStatus =
   | "STARTING"
   | "STARTED"
   | "COMPLETE"
-  | "IN_PROGRESS";
+  | "IN_PROGRESS"
+  | "DELAYED";
```

Comment:
i know i'm being annoying, but why do we have a need to duplicate this type (in `ServicePlanStatus.ts`)? can we dedupe it?

Reply:
that is a good catch, not sure why we don't just import from `ServicePlanStatus`